platform/android/export/export_plugin.cpp

```diff
@@ -1558,9 +1558,10 @@ String EditorExportPlatformAndroid::load_splash_refs(Ref<Image> &splash_image, R
 	}
 
 	// Setup the splash bg color
-	bool bg_color_valid;
-	Color bg_color = ProjectSettings::get_singleton()->get("application/boot_splash/bg_color", &bg_color_valid);
-	if (!bg_color_valid) {
+	Color bg_color;
+	if (ProjectSettings::get_singleton()->has_setting("application/boot_splash/bg_color")) {
+		bg_color = ProjectSettings::get_singleton()->get("application/boot_splash/bg_color");
```

Comment:
Is there a situation where `has_setting` would fail here? It's defined unconditionally in `main.cpp::setup2()`.
And it should likely get the value with `GLOBAL_GET` in case users set a feature override. (E.g. to a have a different bg color on Android from other platforms.)

Reply:
I will change to GLOBAL_GET then.